modules/compiler/spirv-ll/test/spvasm/op_group_i_add.spvasm

```diff
@@ -101,23 +98,6 @@
                                 OpReturn
                                 OpFunctionEnd
 
-; CHCK-LABEL: int_sub_group_reduction
-; CHECK: define spir_kernel void @int_sub_group_reduction(ptr addrspace(1) [[IN:%[a-zA-Z0-9_]+]])
- %int_sub_group_reduction_fcn = OpFunction %void_ty None %int_fcn_ty
-  %int_sub_group_reduction_in = OpFunctionParameter %ptr_CrossWorkgroup_int_ty
-  %int_sub_group_reduction_bb = OpLabel
- ; CHECK: [[VAL:%[a-zA-Z0-9_]+]] = load i32, ptr addrspace(1) [[IN]], align 4
- %int_sub_group_reduction_val = OpLoad %int_ty %int_sub_group_reduction_in Aligned 4
- ; CHECK: [[RES:%[a-zA-Z0-9]+]] = call spir_func i32 @_Z20sub_group_reduce_addi(i32 [[VAL]])
- %int_sub_group_reduction_res = OpGroupIAdd %int_ty %sub_group_scope Reduce %int_sub_group_reduction_val
- ; CHECK: store i32 [[RES]], ptr addrspace(1) [[IN]]
-                                OpStore %int_sub_group_reduction_in %int_sub_group_reduction_res Aligned 4
- ; CHECK: ret void
-                                OpReturn
-                                OpFunctionEnd
-
- ; CHECK: declare spir_func i32 @_Z20sub_group_reduce_addi(i32)
-
```

Comment:
There are a bunch of `CHECK` removals from tests that remove more than I would have expected, is this intentional? Do we no longer generate this at all? Or is this not worth checking, maybe?

Reply:
Ah right I forgot to mention those but yeah, after removing the signedness, test functions like `int_sub_group_reduction` here are the same as `uint_sub_group_reduction` in the same file.

Reply:
In that case maybe `int_sub_group_reduction` can be removed entirely, rather than just taking out the checks?

Reply:
Ah yes you're right I didn't fully complete the job. One sec.

Reply:
That should be that sorted now. Updated the commit message/description too.